File: app/src/main/java/ru/iandreyshev/parserrss/ui/activity/BaseActivity.java
package ru.iandreyshev.parserrss.ui.activity;

import android.widget.Toast;

import com.arellomobile.mvp.MvpAppCompatActivity;

import java.io.Serializable;

import ru.iandreyshev.parserrss.presentation.view.IBaseView;

public abstract class BaseActivity extends MvpAppCompatActivity implements IBaseView {
    @Override
    public final void showShortToast(final String message) {
        Toast.makeText(getApplicationContext(), message, Toast.LENGTH_SHORT).show();
    }

    @Override
    public final void showLongToast(final String message) {
        Toast.makeText(getApplicationContext(), message, Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/kotlin/ru/iandreyshev/parserrss/ui/activity/BaseActivity.kt
package ru.iandreyshev.parserrss.ui.activity

import com.arellomobile.mvp.MvpAppCompatActivity
import org.jetbrains.anko.toast

import ru.iandreyshev.parserrss.presentation.view.IBaseView

abstract class BaseActivity : MvpAppCompatActivity(), IBaseView {
    override fun showToast(message: String) {
        toast(message)
    }
}
